vivisect/parsers/elf.py

```diff
@@ -106,6 +106,7 @@ def loadElfIntoWorkspace(vw, elf, filename=None, baseaddr=None):
     vw.setMeta('DefaultCall', archcalls.get(arch,'unknown'))
 
     vw.addNoReturnApi("*.exit")
+    vw.addNoReturnApi("*.abort")
```

Comment:
oh yes, keep 'em comin.  more where this came from!

Reply:
I'll throw 'em up as I hit em :)